frontend/src/app/inbox/ChatInbox.tsx

```diff
@@ -21,7 +21,7 @@ const ChatInbox = ({ chats, users }: ChatInboxProps) => {
         sortedChats.length > 0 &&
         sortedChats.map((c) => {
             const com = c?.communityConfig || {}
-            return <ChatPreview key={c.id} match={c} users={users} community={com} />
+            return <ChatPreview key={c.id} match={c} users={users} community={com} chatData={c} />
```

Comment:
There is already a property for this, though it is called `match`. If you look in the property list, they both have the value `c`.